tests/test_definitions/multicolumn_map_expectations/expect_multicolumn_sum_to_equal.json

```diff
@@ -35,7 +35,7 @@
       }
     },{
       "title": "row_condition_with_ignore_if_any_are_missing",
-      "suppress_test_for": "spark",
+      "only_for": ["pandas"],
```

Comment:
This seems wrong

Reply:
@jcampbell This is correct, because of the `row_condition` syntax.  We would need to use different syntax for Spark and SQL.  Do you see that?

Reply:
Ah, I see (I coudln't see that part since there was no change on those lines so it was folded). How *did* this test work for sql before though? Is it running?

Reply:
@jcampbell The test worked for SQL for V2 and was not engaged for SQL for V3 because the provider has not yet been implemented.  However, that provider was in the PR state (and is now an approved PR), so I used caution via this fix.  Thanks!